### ISSUE TITLE
请问图片如何进行微调？

### ISSUE THREAD

**Issue by lynnroyal** [InternLM/InternLM-XComposer#6]
如题，这么模型如何进行文字和图片的微调

**Comment by LightDXY** [comment]
Thank you for your attention. The model is pretrained with large-scale image-text pairs and supervised fine-tuned with multi-task datasets and multi-modal instruction tuning datasets, with a simple auto-regressive loss. Please refer to our paper (https://arxiv.org/pdf/2309.15112v2.pdf) for more details.

**Comment by lynnroyal** [comment]
Thx a lot for ur answer, I will read the paper you provided carefully
